modules/snooker/src/main/ts/ephox/snooker/api/TableLookup.ts

```diff
@@ -1,11 +1,12 @@
+import { HTMLTableElement, HTMLTableRowElement, HTMLTableCellElement, Element as DomElement } from '@ephox/dom-globals';
 import { Arr, Fun, Option } from '@ephox/katamari';
 import { Element, Node, SelectorFilter, SelectorFind, Selectors, Traverse } from '@ephox/sugar';
 import { getAttrValue } from '../util/CellUtils';
 import * as LayerSelector from '../util/LayerSelector';
 import * as Structs from './Structs';
 
 // lookup inside this table
-const lookup = function (tags: string[], element: Element, isRoot: (e: Element) => boolean = Fun.never): Option<Element> {
+const lookup = <T extends DomElement = DomElement> (tags: string[], element: Element, isRoot: (e: Element) => boolean = Fun.never): Option<Element<T>> => {
```

Comment:
This is incorrect, as mentioned in the JIRA the return type of this should be `Option<Element<HTMLTableElement>>`

Reply:
Ignore sorry, as I can see it's also used to lookup table rows and you've updated the relevant exported `table` function.